input/pagecontent/index.md

```diff
@@ -1,102 +1,88 @@
 
-## Background
+This IG is a response to this articulation.
 
-TODO
+### Usage
 
-## Motivation
+A `DiagnosticReport` profile with an identifier of `PulmonaryFunctionTestDiagnosticReport` is introduced.
 
-TODO
+This implementation guide can be used by device manufacturers, pulmonary function labratories, and electronic health record vendors to model pulmonary function test data.
+The `PulmonaryFunctionTestDiagnosticReport` defines a set of reference `Observation` Profiles that map to the 2017 [Recommendations for a Standardized Pulmonary Function Report](https://www.thoracic.org/statements/resources/pft/standardized-pulmonary-function-report.pdf). As a design principle, this implementation guide makes use of `derivedFrom` associations for `Observation` Profiles of measures such as the Zscore of a patient's performance against a reference value equation. Where possible, this implementation guide.
 
-(Maybe this can be combined with background.)
+Implementers should consider referencing the examples included as artifacts and in particular [DiagnosticReport-diagnostic-report](./DiagnosticReport-diagnostic-report.html) and its associated Observations.
 
-The objective of this implementation guide is to enable interoperability of pulmonary function test data, allowing health care providers to
+Practical use cases include:
 
+* Defining a target export for the data consolidation process of large research studies on normative pulmonary function
+* Completing digital transformation of paper or portable document format (PDF) versions of pulmonary function test reports to fully electronic health records
+* Connecting and promoting interoperable pulmonary function test records as part of public health research
+* Integrating novel self-surveillance data formats such as mobile at-home spirometry back into a provider's health information system
 
-Motivations/problems solved discussed here are good to include: https://docs.automatemedical.com/docs/pft-extractor-features
+### Limitations
 
-## Usage
+#### Gaps in coding
 
-TODO
+For Observations where an appropriate LOINC or SNOMED code can not be found, we specify a human readable name for the Observation in `code.text`. Missing, ambiguous, or inconsistent LOINC codes/names were encountered during modeling. This problem is known in the existing clinical literature. Successfully progressing this Implementation Guide from its draft state will require co-ordination on recommendations to LOINC or SNOMED to guide resolving gaps in codeable concepts.
 
-## Issues
+A non-exhaustive list of our documentation for these gaps is available in [Coding Gaps](./coding-gaps.html).
```

Comment:
This section ending with the sentence:

>A non-exhaustive list of our documentation for these gaps is available in [Coding Gaps](./coding-gaps.html).

followed by the section heading:

>#### Non-exhaustive

could imply a connection between the two which doesn't exist. The non-exhaustiveness of the Coding Gaps table is unrelated to the non-exhaustiveness of which PFT panels the IG includes.

I like "Non-exhaustive" as a section title, so I think we could refer to the list as "not comprehensive" or "non-comprehensive", or even omit that qualifier here but still say that in `./coding-gaps.html`.

```suggestion
A list of our documentation for these gaps is available in [Coding Gaps](./coding-gaps.html).
```